### ISSUE TITLE
auro-lockup: title has low color contrast

### ISSUE THREAD

**Issue by geoffrich** [AlaskaAirlines/auro-lockup#3]
## Describe the bug
The title's text color uses the `auro-color-brand-breeze-300` token (#00cff0). This has a [contrast ratio of 1.87](https://contrast-ratio.com/#%2300cff0-on-white) on a white background which does not meet WCAG AA color contrast requirements of 4.5 : 1.

## To Reproduce
1. Run color contrast checker on title

## Expected behavior
The color contrast meets WCAG AA requirements.

## Screenshots
![image](https://user-images.githubusercontent.com/4992896/106020847-09673480-6079-11eb-9056-ee9d96d32e61.png)

**Comment by blackfalcon** [comment]
Please see attached PDF. This is the design spec for the lockup. Whereas in the comp, the color specified is `#00C7E6`which is a slight shade darker than `auro-color-brand-breeze-300` token (`#00cff0`)

* `auro-color-brand-breeze-300` Contrast Ratio `1.87:1`
* `#00C7E6` Contrast Ratio `2.03:1`

Both well below the contract rating. 

Suggestion: color spec be updated to use the `var(--auro-color-brand-breeze-400)` token. `#009FB8` Contrast Ratio `3.16:1`. This passes the WCAG AA for large text only. 

See http://lockup-issue3.surge.sh/ for a demo

**Comment by blackfalcon** [comment]
PR created to illustrate options. Email sent to Sr designers to assist in addressing this issue.

**Comment by blackfalcon** [comment]
@vidalmenAS any support on this would be appreciated.
